### ISSUE TITLE
[COGNITION]: Focus should be set on the H1 on every step of scheduling flow

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6070]
## Description

We will want to manage focus out of app flow and in app flow so screen reader users know where they are and what they're being asked to do on each step. For the first view, before users have selected to schedule an appointment, let's not manage focus, and just let the page load normally. For all views in app flow, let's set focus on the `<h1>` so screen readers read intent immediately. Screenshots attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria
**Definition of done**
* [ ] As a screen reader user, I want to hear the `<h1>` read aloud when app pages load
* [ ] As this user, I want the benefit of being oriented on the page when the `<h1>` takes focus on load

## WCAG or Vendor Guidance (optional)

* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-20 at 4 47 09 PM](https://user-images.githubusercontent.com/934879/74987545-07283f00-5401-11ea-9032-29acff2fc565.png)

**Comment by 1Copenut** [comment]
@jbalboni && @lenaecb I retested this in Staging with the following combinations:
* Win10 + IE11 + JAWS
* Win10 + IE11 + Chrome
* Win10 + Firefox + NVDA
* Win10 + Firefox + Chrome
* MacOS + Safari + VoiceOver
* iOS + Safari + VoiceOver Mobile

All worked as expected. Moving this issue to close.
